game-core/src/main/java/games/strategy/triplea/delegate/battle/steps/fire/MarkCasualties.java

```diff
@@ -45,9 +44,7 @@
 
   private final FireRoundState fireRoundState;
 
-  @ChangeOnNextMajorRelease(
-      "returnFire is ALL for everything except NavalBombardment and old saves."
-          + "Rework so that returnFire isn't needed at all.")
+  @RemoveOnNextMajorRelease("returnFire is ALL for everything except old saves")
```

Comment:
side comment, just realized the existing comment does not quite make sense.
Specifically "ALL for everything" reads odd, without context it does not really make sense.
If ALL is an enum or a constant defined somewhere, perhaps fully qualifying it would remove any confusion.